### PR TITLE
Adding Laravel Herd support

### PR DESCRIPTION
**Pull request by nexxai**
Just adding the path of Laravel Herd's PHP binary to the search function

### CODE REVIEW

File: info.plist [hunk 1]
```diff
@@ -83,6 +83,8 @@
     /opt/homebrew/bin/php laravel.php "{query}"
 elif [ -f "/usr/local/bin/php" ]; then
     /usr/local/bin/php laravel.php "{query}"
+elif [ -f "$HOME/Library/Application Support/Herd/bin//php" ]; then
+    "$HOME/Library/Application Support/Herd/bin//php" laravel.php "{query}"
```
Comment (tillkruss) [inline info.plist]: Are the double slashes intentional?

Comment (nexxai) [reply info.plist]: That's exactly what shows up in a "which php"

Comment (tillkruss) [reply info.plist]: Gotcha. And you tested this locally it works?

Comment (nexxai) [reply info.plist]: Yep.  It's what I have setup on my own machine.

**Pull request merged**